apps/wallet/package.json

```diff
@@ -83,6 +83,7 @@
         "@scure/bip32": "^1.1.0",
         "bip39-light": "^1.0.7",
         "bootstrap-icons": "^1.9.1",
+        "browser-passworder": "^2.0.3",
```

Comment:
Looks like this is the updated / maintained package: https://www.npmjs.com/package/@metamask/browser-passworder